src/viadot/orchestration/prefect/tasks/minio.py

```diff
@@ -0,0 +1,63 @@
+"""Task for uploading pandas DataFrame to MinIO."""
+
+import pandas as pd
+from typing import Any, Literal
+
+from prefect import task
+from prefect.logging import get_run_logger
+
+from viadot.sources import MinIO
```

Comment:
I have added contex for this import. Dependencies can be added on a client side (or external repo in general)  because it's not a part of  `viadot-lite`

Reply:
https://github.com/dyvenia/viadot/blob/f5e087046f26c86453ddb091fc643ca27f6e6d12/src/viadot/sources/__init__.py#L34-L38 I think we need here additional condition here. Because now even if we install here minio packeges, source will be not imported still. Could you test if that works in container ? 
`if find_spec("s3fs") or find_spec("minio"):`